integration/v3_grpc_test.go

```diff
@@ -193,3 +194,132 @@ func TestV3DeleteRange(t *testing.T) {
 		clus.Terminate(t)
 	}
 }
+
+// TestV3WatchFromCurrentRevision tests Watch APIs from current revision.
+func TestV3WatchFromCurrentRevision(t *testing.T) {
+	testKey, testKeyL, testKey2 := "foo", "fooLong", "helloworld"
```

Comment:
this is pretty confusing. I feel we should just embed the key into the test cases. Then it is much more readable.